configs/milvus.yaml

```diff
@@ -176,10 +176,12 @@ queryNode:
       pulsarBufSize: 512 # pulsar channel buffer size
     searchResult:
       recvBufSize: 64 # msgPack channel buffer size
-  # Segcore will divide a segment into multiple chunks.
+  # Segcore will divide a segment into multiple chunks to enbale small index
   segcore:
     chunkRows: 32768 # The number of vectors in a chunk.
-
+    smallIndex:
+      nlist: 256 # small index nlist, recommend to set sqrt(chunkRows), must smaller than chunkRows/8
+      nprobe: 16 # nprobe to search small index, based on your accuracy requirement, must smaller than nlist
```

Comment:
in general, we need not limit "nprobe <= nlist", we can handle it in source code like this:
nprobe = std::min(nprobe, nlist)
Just like how faiss handle index parameters for IVF

Reply:
it is already handled by go side

	defaultNprobe := p.SmallIndexNlist / 16
	p.SmallIndexNProbe = p.Base.ParseInt64WithDefault("queryNode.segcore.smallIndex.nprobe", defaultNprobe)
	if p.SmallIndexNProbe > p.SmallIndexNlist {
		log.Warn("small index nprobe must smaller than nlist, force set to", zap.Any("nprobe", p.SmallIndexNlist))
		p.SmallIndexNProbe = p.SmallIndexNlist
	}